cyral/provider.go

```diff
@@ -171,6 +170,7 @@ func Provider() *schema.Provider {
 			"cyral_repository_datamap":               resourceRepositoryDatamap(),
 			"cyral_repository_identity_map":          resourceRepositoryIdentityMap(""),
 			"cyral_repository_local_account":         resourceRepositoryLocalAccount(),
+			"cyral_repository_user_account":          resourceRepositoryUserAccount(),
```

Comment:
I think we should deal with "database accounts" instead of "user accounts" (at least in user-facing stuff), because that's what we're calling user accounts in the UI.

Reply:
Ah, I see!

Reply:
I think I am going to keep it matching the API. From @wcmjunior:
`Historically our UI has some influences that go well beyond the engineering team, meaning that it can change over time. If something is called Database account today it could be changed to something else in the future due to some UI change for some specific new workflow/feature/other requirement and the API could remain untouched.`